accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-input-rules.ts

```diff
@@ -78,7 +78,8 @@ let a11yRulesInput: Rule[] = [
             if (textTypes.indexOf(type) !== -1) { // If type is in the list
                 // Get only the non-hidden labels for element, in the case that an label is hidden then it is a violation
                 // Note: label[for] does not work for ARIA-defined inputs
-                let labelElem = ruleContext.hasAttribute("role") ? null : RPTUtil.getLabelForElementHidden(ruleContext, true);
+                //let labelElem = ruleContext.hasAttribute("role") ? null : RPTUtil.getLabelForElementHidden(ruleContext, true);
+                let labelElem = RPTUtil.getLabelForElementHidden(ruleContext, true);
```

Comment:
label/for is only valid for HTML native elements. When `role` is specified, this form of labelling is not supported - they must use the `aria-label` / `aria-labelledby` pattern. Original code was correct.

Reply:
didn't realize this.  restored.